setup.py

```diff
@@ -23,6 +23,11 @@
 
 # Read requirements
 install_reqs = parse_requirements(os.path.join(here, 'requirements.txt'), session='hack')
+# for pip 20.2.1
+try:
+    requirements = [str(ir.req) for ir in install_reqs]
+except Exception:
+    requirements = [str(ir.requirement) for ir in install_reqs]
```

Comment:
```suggestion
try:
    requirements = [str(ir.requirement) for ir in install_reqs]
    
except Exception:  # for pip < 20.1
    requirements = [str(ir.req) for ir in install_reqs]
```

it has possibly be broken by https://github.com/pypa/pip/pull/7747, shipped in pip 20.1

let's put the modern way first, more future proof